exercise3.js

```diff
@@ -0,0 +1,10 @@
+// 3) Print out every third number between 1 and 100. Try to limit the number of iterations (it should be less than 100). 
+
+for ( let i = 1; i <=100; i+=3 ) {
+   
+    if( i === 1) {
```

Comment:
This condition is not necessary. With current code you print:
```
4
7
10
...
```
So you actually miss the `1` that should also be here. And even if you wanted to omit it, it's better to just start the for-loop from 4 (`let i = 4`) 😄